app/controllers/participant_signup_controller.rb

```diff
@@ -0,0 +1,33 @@
+class ParticipantSignupController < ApplicationController
+	before_filter :fetch_participant
+  	skip_before_filter :check_authorization	
+
+
+	def index
+	  redirect_to :action => :participant_signup_intake_form
+	end
+
+	def intake_form
+	  unless current_user && current_user.person.can_view?(@participant)
```

Comment:
I think there should be a check that `current_user.person` is a `Participant`, rather than if they can view the `Participant`.